canonical-websites/snapcraft.io
# Build all bundles with webpack

Fixes #1227
Fixes #1241 
Fixes #1242

Moves build of all bundles from rollup to webpack, removes rollup and related dependencies, adds back `watch-js` with webpack.

### QA
- ./run build
- all JS files should build with webpack
- ./run or demo
- test all the pages with scripts: store featured snaps, snap details channel map and screenshots, publisher pages (listing, measures, etc)
- everything should work as before
- ./run exec yarn watch-js
- webpack should run in watch mode and rebuild JS bundles when any file changes
